File: code/storage/EletricGo/Domain/Storages/StorageDto.cs
using EletricGo.Migrations;

namespace EletricGo.Domain.Storages
{
    public class StorageDto
    {
        public Guid Id { get; set; }
        public string Designation { get; set; }
        public Location Location { get; set; }

        public List<ChargingSystem> ChargingSystems { get; set; }

        public StorageDto(Guid id, string designation, Location location, List<ChargingSystem> chargingSystems)
        {
            Id = id;
            Designation = designation;
            Location = location;
            ChargingSystems = chargingSystems;
        }
    }
}

File: code/spa/src/services/dtos/StorageDTO.ts
import type ChargingSystems from "@/models/chargingSystem";

export default interface StorageDTO {
    id: string;
    designation: string;
    //location: Location;
    location: 
    {
      latitude: string,
      longitude: string,
      altitude: string,
      address: {
        street: string,
        door: string,
        floor: string,
        postalCode: string
        city: {
            number: number,
            name: string
        }
      }
    }
  ,
    chargingSystems: Array<ChargingSystems>;



}


